src/breaker.js

```diff
@@ -58,18 +62,27 @@ function breaker(config, emitter) {
         active = true;
         emitter.emit('circuitBroken', status(ttl));
         timer = setTimeout(restoreCircuit, ttl);
-      }
-      else {
+      } else {
         setTimeout(decreaseViolation, config.breaker.toleranceFrame);
       }
     }
   }
 
   function status(ttl) {
-    return { step, active, ttl };
+    return {step, active, ttl};
   }
 
-  return { circuitError, openCircuit, restoreCircuit, closeCircuit, status };
+  return {
+    circuitError,
+    openCircuit,
+    restoreCircuit,
+    closeCircuit,
+    status,
+
+    // TODO DPL: Deprecate this contract and remove duplicated functions (also, `close` === `restore`?)
```

Comment:
Restore is an attempt to close an open circuit. If the attempt fail, we re-open the circuit, but increment the step so that it's open longer. The minute that we get a successful reply in a close attempt, it closes the circuit for good

Reply:
ok, so restore is like a `next` or `untrip` in our case?